### DISCUSSION TITLE
Packages/com.unity.entities/manual/transforms-using

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#567] [Notes]
# Packages/com.unity.entities/manual/transforms-using

https://docs.unity3d.com/Packages/com.unity.entities@latest/index.html?subfolder=%2Fmanual%2Ftransforms-using.html

**Comment by vertxxyz** [comment]
When you set the `Parent` component, the parent entity must have a `LocalToWorld` component or else the `ParentSystem` will silently remove the child.
